python_ta/__init__.py

```diff
@@ -166,6 +166,24 @@ def _load_config(linter, config_location):
     linter.load_config_file()
 
 
+def _load_messages_config(path: str, def_path: str) -> dict:
```

Comment:
Since `def` has a different meaning in Python (and programming in general), it's not a good short form for `default`. Please change `def_path` to `default_path` both here and in `messages_config_def_path`.